ansible/roles/ruby-common/tasks/main.yml

```diff
@@ -67,10 +67,24 @@
     - rake-env
     - ruby-gc-wrapper
 
+- name: Uninstall Bundler
+  gem:
+    name: bundler
+    state: absent
+    user_install: no
+  sudo: true
+  register: out
+
 - name: Install Bundler
-  command: "{{ ruby_location }}/bin/gem install bundler {{ ruby_bundler_flags }}
-            creates={{ ruby_location }}/bin/bundle"
+  gem:
+    name: bundler
+    version: "{{ bundler_version }}"
+    state: present
+    user_install: no
   sudo: true
+  register: out
+
+- debug: var=out
```

Comment:
should this remain in? i have no idea what it does, but it looks an awful lot like debug code for some reason